indico/modules/events/registration/util.py

```diff
@@ -999,11 +1000,9 @@ def get_persons(registrations, include_accompanying_persons=False):
     return persons
 
 
-def process_uploaded_registration_picture(source, size):
-    """Process the uploaded registration picture.
-
-    This converts the picture to JPEG and resizes it to a maximum size.
-    """
+def process_registration_picture(source, thumbnail=False):
+    """Resizes the picture to a maximum size and saves it to JPEG."""
+    max_size = REGISTRATION_PICTURE_SIZE if not thumbnail else REGISTRATION_PICTURE_THUMBNAIL_SIZE
```

Comment:
```suggestion
    max_size = REGISTRATION_PICTURE_THUMBNAIL_SIZE if thumbnail else REGISTRATION_PICTURE_SIZE
```

no need for that negation